plugins/LadspaEffect/caps/interface.cc

```diff
@@ -121,11 +109,8 @@ void caps_so_init()
 
 	/* make sure N is correct */
 	assert (d - descriptors == N);
-
-	//seed();
```

Comment:
@Rossmaxx can you explain the reasoning behind this minor formatting change?  Is it intended to be upstreamed?

Reply:
My intention here was to remove the commented out code. Will look again.

Reply:
Right, but according to the diff, I believe this comment was placed there by the author, Tim Goetze, so we should probably leave it there.

Reply:
Oh, I see.  If `seed()` needs that header, removing the commented code does makes sense.

Reply:
Ok, I can see the removed (commented) `seed()` definition too.  Hmm... Yeah I'm not sure the author's intent there, but I agree, this is more consistent.  When we submit this upstream, the author may reject this portion.  It may be less intrusive to comment the header as you've mentioned, but I think it's fine either way.

Reply:
Perhaps:


```diff 
- #include <sys/time.h>
+ // Needed only for seed(), which is disabled
+ // #include <sys/time.h>
```

Reply:
I think a diff like this, but combined with reversion of commented code would give the author of the project the option to remove this themselves.

Reply:
fixed this.